src/chrome/content/rules/CapitalOne.com.xml

```diff
@@ -52,18 +47,13 @@ Fetch error: http://travelrewardspn.capitalone.com/ => https://travelrewardspn.c
 	<target host="jobs.capitalone.com" />
 	<target host="loanhub.capitalone.com" />
 	<target host="loanhub2.capitalone.com" />
-	<target host="login.capitalone.com" />
-	<target host="login1.capitalone.com" />
-	<target host="login2.capitalone.com" />
```

Comment:
Maybe a valid test url works for them, e.g. `http://login.capitalone.com/resources/images/icon_padlock.gif`